pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -668,6 +668,7 @@ enum ContainerState {
     CONTAINER_RUNNING = 1;
     CONTAINER_EXITED  = 2;
     CONTAINER_UNKNOWN = 3;
+    CONTAINER_OOMKILLED = 4;
```

Comment:
Don't think `OOMKILLED` should be a state. It's just a reason of Exited.

Actually I don't know why [kuberuntime needs to use `OOMKilled` directly](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/kuberuntime_container.go#L393).

Can't it be reflected in exit code?